### DISCUSSION TITLE
touch screen behaviour

### DISCUSSION THREAD

**Discussion by emdete** [fyne-io/fyne#3521] [Q&A]
i develop for the Librem5 which is a phone with a touchscreen and should behave like one expects, for example wiping should scroll through a list. But it doesnt. Touches are recognized as clicks. Can i somhow tell Fyne to do a usual touchscreen behaviour?

**Comment by Jacalz** [comment]
Assuming that you are using the phone as a phone, are you building using `-tags mobile`?
Doing so should enable the mobile driver and perhaps work a bit better for you.

**Reply by emdete** [reply]
i didnt know about tags until i read the Wayland discussion here - are these documented somewhere?

**Reply by Jacalz** [reply]
They are available here: https://developer.fyne.io/explore/compiling
I notice that it is missing the `wayland` tag but that likely is because it isn't fully supported yet.

**Reply by emdete** [reply]
i tried that but the behaviour is strange, touching a single item scrolls the list a bit before selecting so i am not able to select the item i want and dragging selects an item as well. so dragging is generally working but has issues i would say. shall i do a bugreport?

**Reply by emdete** [reply]
https://developer.fyne.io/explore/compiling is missing wayland. are there even more?

**Reply by Jacalz** [reply]
> https://developer.fyne.io/explore/compiling is missing wayland.

Indeed. See my response above about that.

> are there even more?

Not ones that we officially support in Fyne, no. Some of our dependencies could technically have further build tags but is outside of our scope unless they are especially useful for us.

**Reply by Jacalz** [reply]
If something isn't working as expected, you can definitely open a bug report :)
